zanata-war/src/test/java/org/zanata/file/TranslationDocumentUploadTest.java

```diff
@@ -23,14 +23,13 @@
 import static javax.ws.rs.core.Response.Status.NOT_ACCEPTABLE;
 import static org.mockito.Mockito.doThrow;
 
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
 import org.mockito.Mock;
 import org.mockito.MockitoAnnotations;
-import org.testng.annotations.AfterMethod;
-import org.testng.annotations.BeforeMethod;
-import org.testng.annotations.Test;
 import org.zanata.exception.ChunkUploadException;
 
-@Test(groups = { "unit-tests" })
```

Comment:
On 2015-06-02 14:11, Carlos Munoz wrote:
> In
> zanata-war/src/test/java/org/zanata/file/TranslationDocumentUploadTest.java
> <https://github.com/zanata/zanata-server/pull/823#discussion_r31491869>:
> 
>>  import org.zanata.exception.ChunkUploadException;
>>  
>> -@Test(groups = { "unit-tests" })
> 
> In general, I guess we are losing test groups. I take it we were not
> using them.

No, we weren't really using them any more, except to separate unit tests
from integration tests.  I think file naming conventions will suffice,
but we could use JUnit's Categories if we want finer control.

> 
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zanata/zanata-server/pull/823/files#r31491869>.
> 


-- 
Sean Flanigan

Principal Software Engineer
Localisation Tools Engineering
Red Hat